src/components/App.jsx

```diff
@@ -0,0 +1,11 @@
+import Calculator from './Calculator';
+
+function App() {
```

Comment:
- [ ] Kindly make this component a class based component instead of a functional one, just like the calculator.js